Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -147,19 +147,27 @@ class DataDecoderTask
     };
 
     mDecoder->reset();
-    decodeTF(pc);
     for (auto&& input : pc.inputs()) {
+      //std::cout << "input: " << input.spec->binding << std::endl;
       if (input.spec->binding == "readout") {
         decodeReadout(input);
       }
+      if (input.spec->binding == "TF") {
+        decodeTF(pc);
+      }
     }
+    mDecoder->computeDigitsTime();
 
     auto& digits = mDecoder->getOutputDigits();
     auto& orbits = mDecoder->getOrbits();
+    std::set<OrbitInfo> ordered_orbits(orbits.begin(), orbits.end());
```

Comment:
that's the same code as in dumpOrbit lambda in DataDecoder.cxx, right ? Could reuse it if the lambda there is actually transformed into a (free) function

Reply:
Done.